.github/workflows/ci.yml

```
@@ -391,6 +393,54 @@ jobs:
           name: webapp-client
           path: webapp/client/
 
+
+  devolutions-gateway-players:
```

suggestion: The project is "player".

```suggestion
  devolutions-gateway-player:
```